src/groups/general_linear.jl

```diff
@@ -246,6 +246,15 @@ project(::GeneralLinear, p, X) = X
 project!(::GeneralLinear, q, p) = copyto!(q, p)
 project!(::GeneralLinear, Y, p, X) = copyto!(Y, X)
 
+function Random.rand!(G::GeneralLinear, pX; kwargs...)
```

Comment:
Would that maybe be a generic good idea for certain manifolds (isometrically embedded ones or such?) to pass rand to the embedding?

Reply:
I would prefer to avoid trait dispatch here to keep the code simple.

Reply:
Sure, that is also fine :)